src/ejercicios.js

```diff
@@ -29,8 +30,9 @@ function btnConectar() {
     db.connect();
     Utils.enableAll();
```

Comment:
Esta línea ya está en el `finally`.
```suggestion
    Utils.enableAll();
```